src/chains/opStack/decorators/walletL2.ts

```diff
@@ -0,0 +1,84 @@
+import type { Client } from '../../../clients/createClient.js'
+import type { Transport } from '../../../clients/transports/createTransport.js'
+import type { Account } from '../../../types/account.js'
+import type { Chain } from '../../../types/chain.js'
+import {
+  type InitiateWithdrawalParameters,
+  type InitiateWithdrawalReturnType,
+  initiateWithdrawal,
+} from '../actions/initiateWithdrawal.js'
+
+export type WalletActionsL2<
+  chain extends Chain | undefined = Chain | undefined,
+  account extends Account | undefined = Account | undefined,
+> = {
+  /**
+   * Initiates a [withdrawal](https://community.optimism.io/docs/protocol/withdrawal-flow/#withdrawal-initiating-transaction) on an L2 to the L1.
```

Comment:
```suggestion
   * Initiates a {@link https://community.optimism.io/docs/protocol/withdrawal-flow/#withdrawal-initiating-transaction withdrawal} on an L2 to the L1.
```
Curious if there is a benifit to doing it with markdown isntead of jsdoc

Reply:
Think they both format the same in IDE.